packages/analytics/analytics-utilities/src/types/chart-types.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+export enum ChartTypes {
+  DOUGHNUT = 'Doughnut',
+  HORIZONTAL_BAR = 'HorizontalBar',
+  VERTICAL_BAR = 'VerticalBar',
+  LINE = 'Line',
```

Comment:
Just curious if this is intentional, eg: do we want to revert here from using `TimeSeriesLine`?

Reply:
Ya my bad this needs to be TimeseriesLine = Line